docs/scripting-ref.rst

```diff
@@ -149,6 +149,24 @@ Enable or disable browser plugins (e.g. Flash).
 Plugins are disabled by default.
 
 
+.. _splash-request-body-enabled:
+
+splash.request_body_enabled
+----------------------------
+
+Enable or disable storage of request content.
+
+**Signature:** ``splash.request_body_enabled = true/false``
+
+By default Splash doesn't keep bodies of each request in memory. It means that
+request content is not available in :ref:`splash-request-info` and in HAR_
+exports. To make request content available to a Lua script set
+``splash.request_body_enabled = true``.
+
+Note that request data in :ref:`splash-request-info` is not available in the
+callback :ref:`splash-on-response-headers` or in the request of the response
+returned by :ref:`splash-http-get` and :ref:`splash-http-post`.
```

Comment:
Where is it available then? You mentioned HAR, is it available in on_request callbacks?

Reply:
Yes, and also in on_response (in response.request.info).

I was going to add a test for that now, but I hit what I believe to be a bug, which I reported as #814

Reply:
Now it also has a test for on_response.